install-dev/init.php

```diff
@@ -50,7 +50,9 @@
 
 require_once _PS_CORE_DIR_.'/config/defines.inc.php';
 require_once _PS_CORE_DIR_.'/config/autoload.php';
-require_once _PS_CORE_DIR_.'/config/bootstrap.php';
+if (file_exists(_PS_CORE_DIR_.'/app/config/parameters.yml')) {
```

Comment:
yes, but in the upgrade.php, bootstrap.php is included in init.php before the creation of the parameters.yml.
To be able to properly include the bootstrap.php file once the parameters.yml is created by Migrate::migrateSettingsFile() I need to have this check.